<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QSplitter" name="splitter_in">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGroupBox" name="groupBox_bracket">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Bracket</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_bucket">
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>25</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
       </layout>
      </widget>
      <widget class="QSplitter" name="splitter_out">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QGroupBox" name="groupBox_step">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>150</height>
         </size>
        </property>
        <property name="title">
         <string>Step</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_step">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>25</number>
         </property>
         <property name="rightMargin">
          <number>10</number>
         </property>
         <property name="bottomMargin">
          <number>10</number>
         </property>
         <item>
          <widget class="QGroupBox" name="groupBox_toolbar">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string/>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_toolbar">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QGroupBox" name="GroupBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="label_type">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="lineWidth">
                 <number>2</number>
                </property>
                <property name="text">
                 <string>None</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_package">
                <property name="text">
                 <string/>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="button_studioPipe">
             <property name="text">
              <string/>
             </property>
             <property name="default">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTreeWidget" name="treeWidget">
             <property name="columnCount">
              <number>0</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QTextEdit" name="textEdit_output"/>
      </widget>
      <widget class="QGroupBox" name="groupBox_details">
       <property name="title">
        <string>Details</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_details">
        <property name="spacing">
         <number>5</number>
        </property>
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>25</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>685</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_aboutApplication"/>
   </widget>
   <addaction name="menu_help"/>
  </widget>
  <action name="action_removeThumbs">
   <property name="text">
    <string>Remove Thumbs</string>
   </property>
  </action>
  <action name="action_removePYC">
   <property name="text">
    <string>Remove PYC</string>
   </property>
  </action>
  <action name="actionPatch">
   <property name="text">
    <string>lsDf</string>
   </property>
  </action>
  <action name="action_minor">
   <property name="text">
    <string>Minor</string>
   </property>
  </action>
  <action name="action_major">
   <property name="text">
    <string>Major</string>
   </property>
  </action>
  <action name="actionLish">
   <property name="text">
    <string>lish</string>
   </property>
  </action>
  <action name="action_aboutApplication">
   <property name="text">
    <string>About application</string>
   </property>
  </action>
  <action name="action_new">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="action_patch">
   <property name="text">
    <string>Patch</string>
   </property>
  </action>
  <action name="action_addon">
   <property name="text">
    <string>Add-On</string>
   </property>
  </action>
  <action name="action_update">
   <property name="text">
    <string>Update</string>
   </property>
   <property name="toolTip">
    <string>Update</string>
   </property>
  </action>
  <action name="action_remove">
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
  <action name="action_updateAll">
   <property name="text">
    <string>Update All</string>
   </property>
  </action>
  <action name="action_reload">
   <property name="text">
    <string>Reload</string>
   </property>
  </action>
  <action name="action_treeDisplay">
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Tree Display</string>
   </property>
  </action>
  <action name="action_groupDisplay">
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Group Display</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
